joomla/joomla-cms
# MSSQL Replace datetime to datetime2(0) type for all joomla columns.

Pull Request for Issue #11277

### Summary of Changes

Change type for 69 columns from type datetime to datetime2(0).
A few indexes change name. 
Sql updates files recreated indexes after change type.

I did a real test for "sql update files" from joomla 3.5 and upgrade to currrent PR (with PR #14102).

### Testing Instructions
* Install joomla with applied patch (no sample data).
* Create a few articles without date on publish down. 
* Check on back end or home page if they are expired or not.
Before patch they are expired, after patch it is correct.

### Expected result
Published articles  without publish_down date appears as not expired.

### Actual result
Published articles  without publish_down date appears as expired.

### Documentation Changes Required
None
